[quant][fix] Fix the call to get_executorch_backend_config

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #86338

Summary:
previously the call failed because there was an infinite loop in _get_share_qparams_ops_configs

Test Plan:
python test/test_quantization.py -k test_get_executorch_backend_config

Reviewers:

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
[quant][fix] Fix the call to get_executorch_backend_config

Summary:
previously the call failed because there was an infinite loop in _get_share_qparams_ops_configs

Test Plan:
python test/test_quantization.py -k test_get_executorch_backend_config

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

andrewor14 reviewed (approved):
Oops... thanks for fixing this!

jerryzh168 commented:
@pytorchbot rebase

jerryzh168 committed:
Update on "[quant][fix] Fix the call to get_executorch_backend_config"


Summary:
previously the call failed because there was an infinite loop in _get_share_qparams_ops_configs

Test Plan:
python test/test_quantization.py -k test_get_executorch_backend_config

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot merge -g

jerryzh168 commented:
@pytorchbot rebase

jerryzh168 committed:
Update on "[quant][fix] Fix the call to get_executorch_backend_config"


Summary:
previously the call failed because there was an infinite loop in _get_share_qparams_ops_configs

Test Plan:
python test/test_quantization.py -k test_get_executorch_backend_config

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot rebase

jerryzh168 committed:
Update on "[quant][fix] Fix the call to get_executorch_backend_config"


Summary:
previously the call failed because there was an infinite loop in _get_share_qparams_ops_configs

Test Plan:
python test/test_quantization.py -k test_get_executorch_backend_config

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot merge -g
